lib/logstash/outputs/elasticsearch/data_stream_support.rb

```diff
@@ -123,18 +143,28 @@ def assert_es_version_supports_data_streams
     DATA_STREAMS_ENABLED_BY_DEFAULT_LS_VERSION = '8.0.0'
 
     # when data_stream => is either 'auto' or not set
-    def data_stream_default(data_stream_params, valid_data_stream_config)
-      ds_default = Gem::Version.create(LOGSTASH_VERSION) >= Gem::Version.create(DATA_STREAMS_ENABLED_BY_DEFAULT_LS_VERSION)
+    # @param data_stream_params [#any?]
+    # @param invalid_data_stream_config [#any?#inspect]
+    def data_stream_default(data_stream_params, invalid_data_stream_config)
```

Comment:
I'm wondering if it makes sense to log whether or not we're using data streams at `info` rather than `debug`, when using `auto` - it's new behaviour and it may be surprising to users. WDYT?

Reply:
Resolved in 311b1dd